Snippets/MsmqTransport/MsmqTransport_1/DoNotCreateQueuesOnInstall.cs

```diff
@@ -0,0 +1,14 @@
+﻿using NServiceBus;
+
+class DoNotCreateQueuesOnInstall
+{
+    DoNotCreateQueuesOnInstall(EndpointConfiguration endpointConfiguration)
+    {
+        #region do-not-create-queues-on-install
+
+        var transport = endpointConfiguration.UseTransport<MsmqTransport>();
+        transport.DoNotCreateQueuesOnInstall();
```

Comment:
Have we considered switching to not create by default ? (and make creation opt-in)

Reply:
By default the installers are not run. Therefore the default behavior is that these queues will not created. Queues are only created if the `EnableInstallers` API is called. This API was named explicitly so its clear to the user that queues will not be created even if EnableInstallers() is being called.